### PR TITLE
Set alert to iOS notifications

### PR DESCRIPTION
**Pull request by mstmustisnt**

### CODE REVIEW

File: lib/push-notifications.js [hunk 1]
```diff
@@ -121,12 +121,10 @@ PushNotifications.prototype.send = function (pushId, data, callback) {
     messageAPN = new apn.Notification();
     _.extend(messageAPN, _this.settings.apn.defaultData);
     messageAPN.expiry = Math.floor(Date.now() / 1000) + _this.settings.apn.expiry;
-    messageAPN.badge = data.badge || _this.settings.apn.badge;
-    messageAPN.sound = data.sound || _this.settings.apn.defaultData.sound;
-    messageAPN.alert = data.title;
-    messageAPN.category = data.category;
-    delete data.title;
```
Comment (mstmustisnt) [inline lib/push-notifications.js]: everything directly in data is undefined except from `data.data`

File: lib/push-notifications.js [hunk 2]
```diff
@@ -55,6 +55,7 @@ PushNotifications.prototype.send = function (pushId, data, callback) {
   var apnOptions = {};
   var admOptions = {};
 
+  var gsmData = {};
```
Comment (vedi) [inline lib/push-notifications.js]: `var gsmData;`

and see below

File: lib/push-notifications.js [hunk 3]
```diff
@@ -88,10 +89,10 @@ PushNotifications.prototype.send = function (pushId, data, callback) {
 
   // Android GCM
   if (regIdsGCM[0] != undefined) {
-    data = _.extend({
```
Comment (vedi) [inline lib/push-notifications.js:91]: `if (!gsmData) {`...

Comment (mstmustisnt) [reply lib/push-notifications.js:91]: if you could explain why we need this change. To avoid storing empty object in memory if no android devices? or why

Comment (vedi) [reply lib/push-notifications.js:91]: In order to do this only 1 time. Am I wrong?

**Pull request merged**